spyder/plugins/configdialog.py

```diff
@@ -317,7 +317,12 @@ def load_from_conf(self):
             radiobutton.toggled.connect(lambda _foo, opt=option:
                                         self.has_been_modified(opt))
         for lineedit, (option, default) in list(self.lineedits.items()):
-            lineedit.setText(self.get_option(option, default))
+            option_value = self.get_option(option, default)
+            if isinstance(option_value, tuple):

```

Comment:
In the `set_columns` method of `EdgeLine` you're transforming all values saved to `edge_line_columns` to lists, but here you're checking only for tuples.

So which one is right? Tuples or lists? :-)